docs/content/guides/limiting-concurrency-in-data-pipelines.mdx

```diff
@@ -0,0 +1,471 @@
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <strong>Run</strong>
+      </td>
+      <td>A run is a single execution of a job in Dagster.</td>
+    </tr>
+    <tr>
+      <td>
+        <strong>Run queue</strong>
+      </td>
+      <td>A sequence of Dagster runs waiting to be executed.</td>
+    </tr>
+    <tr>
+      <td>
+        <strong>
+          <a href="/deployment/executors">Executor</a>
+        </strong>
+      </td>
+      <td>
+        Executors are responsible for executing steps within a job run.
+        Executors can range from single-process serial executors to managing
+        per-step computational resources with a control plane.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+### Understanding Dagster’s concurrency limit options
+
+Dagster supports placing limits on the number of processes that can be in progress at a single time, at the run and op-level. Depending on your needs, one or both types may be suitable:
+
+- **Run concurrency** controls the total number of runs in a deployment that can execute at the same time. Run concurrency applies to both op and asset-based jobs and all [code locations][code-location-concept] in a single deployment. Any runs beyond the limit will be queued and won’t use any compute.
+- **Op concurrency** controls the number of ops that can execute in parallel **in a single run**. **Note**: Op-level concurrency is a bit misleading - this type of concurrency applies to both op **and** asset-based jobs.
+
+By controlling the number of parallel processes Dagster executes, you can ensure that resources are not overwhelmed and that each process has the resources it needs to run efficiently. This can lead to faster and more reliable pipeline execution, as well as easier monitoring and debugging of issues.
+
+---
+
+## Configuring run-level concurrency
+
+Run concurrency can be configured by:
+
+- Limiting the overall number of runs in a deployment
+- Specifying limits using tags, which is useful for applying limits to specific jobs. For example, jobs that share a resource could all have the same tag.
+
+These methods of limiting concurrency can be used individually or together. For each queued run, Dagster will check that launching the run will satisfy all conditions. For example, if launching a run would exceed the maximum number of concurrent runs, Dagster will queue the run until another run finishes.
+
+**Note**: Setting maximum concurrent runs for the same job isn't currently supported. If you want to prevent a new run from launching if a current run is in progress, we recommend writing some custom logic in a job schedule. Check out [this GitHub discussion for an example](https://github.com/dagster-io/dagster/discussions/8414).
+
+<TabGroup>
+<TabItem name="Limiting overall runs">
+
+### Limiting overall runs
+
+How you limit the overall number of runs in a deployment depends on whether you're using Dagster Cloud or Dagster Open Source:
```

Comment:
nitpick but maybe "Limiting the overall number of *in-progress* runs or *concurrent* runs?